problemsets-mat223/modules/module15-exercises.tex

```diff
@@ -1,17 +1,26 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
-		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
+		\prob  	For each linear transformation defined below, find the eigenvectors and eigenvalues of it.
+    		\begin{enumerate}
+    			\item   $\mathcal S:\R^2\to\R^2$, where $\mathcal S$ is a linear transformation that stretch every vector by the factor of $3$.
+    			\item   $\mathcal R:\R^2\to\R^2$, where $\mathcal R$ a linear transformation that rotates every vector clockwise by $\frac{\pi}{4}$.
+    			\item   $\mathcal P:\R^2\to\R^2$, where $\mathcal P$ a linear transformation that projects every vector onto the line $\ell$ given by $y=-x$.
+    			\item   $\mathcal F:\R^2\to\R^2$, where $\mathcal F$ is a linear transformation that reflects every vector over the line $ell$ given by $y=-x$.
+    			\item   $T:\R^3\to\R^3$, where $T$ is a linear transformation induced by the matrix $M_T = \mat{1&2&3\\3&4&5\\5&6&7}$.
+    			\item   $U:\R^3\to\R^2$, where $U$ is a linear transformation induced by the matrix $M_U = \mat{1&2&3\\3&4&5}$.
+    		\end{enumerate}
+    		
+		\prob Let $A = \mat{a&b\\c&d}$, where $a,b,c,d \in R$.
+		    \begin{enumerate}
+		        \item   If $ad > bc$, then find the characteristic polynomial of $A$.
```

Comment:
The characteristic polynomial is always the same regardless of this relationship. Do you mean, find the eigenvalue(s)?